### PR TITLE
Add masti movies

### PR DESCRIPTION
**Pull request by sanjeevkpandit**
Add masti movie

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -88,10 +88,13 @@ A list of movies/series which could have really interesting porn parody names.
 | Chak de! India | Chod de! India |
 | Dum Laga Ke Haisha | :stuck_out_tongue_winking_eye: |
 | Entertainment | Enter Ten Men |
+| Grand Masti | :stuck_out_tongue_winking_eye: |
+| Great Grand Masti | :stuck_out_tongue_winking_eye: |
 | Jab We Met | Jab We Mate |
 | Kabhi Khusi Kabhi Gum | Kabhi Pussy Kabhi Bum |
 | Kuch Kuch Hota Hai | Kuch Kuch Chota Hai |
 | Love Sex Aur Dhokha | :stuck_out_tongue_winking_eye: |
+| Masti | :stuck_out_tongue_winking_eye: |
```
Comment (aviskarkc10) [inline README.md]: maybe we can just use this one and remove the sequels

Comment (aviskarkc10) [reply README.md]: @mesaugat @SafalPandey ??

Comment (mesaugat) [reply README.md]: Yep.

Comment (sanjeevkpandit) [reply README.md]: Updated.

**Pull request merged**